### PR TITLE
Release v1.37.0

### PR DESCRIPTION
**Pull request by torredil**
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by ConnorJC3 — COMMENTED** [review]
/lgtm
/approve

**Review by AndrewSirenko — APPROVED** [review]

**Review by ElijahQuinones — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+*(No, really, you MUST read this before you upgrade)*
+
+#### [ACTION REQUIRED] Update to the EBS CSI Driver IAM Policy
+Due to an upcoming change in handling of IAM polices for the CreateVolume API when creating a volume from an EBS snapshot, a change to your EBS CSI Driver policy may be needed. For more information and remediation steps, see [GitHub issue #2190](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/2190). This change affects all versions of the EBS CSI Driver and action may be required even on clusters where the driver is not upgraded.
+
+### Notable Changes
+* Export EBS detailed performance statistics as Prometheus metrics for CSI-managed volumes ([#2216](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2216), [@torredil](https://github.com/torredil))
+
+### Bug Fixes
+* Update example-iam-policy.json, ([#2193](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2193), [@willswire](https://github.com/willswire))
```
Comment (AndrewSirenko) [inline CHANGELOG.md:12]: Nit: Lingering comma, and consider adding "for non 'aws' partitions"

**Review by AndrewSirenko — APPROVED** [review]

**Comment by ConnorJC3** [comment]
Performing manual override merge due to defect in kustomize tests

**Pull request merged**